tests/crypto/secp256r1/src/main.c

```diff
@@ -21,11 +21,6 @@
 #else /* CONFIG_MBEDTLS_PSA_P256M_DRIVER_RAW */
 #include "psa/crypto.h"
 #endif /* CONFIG_MBEDTLS_PSA_P256M_DRIVER_RAW */
-#else /* CONFIG_MBEDTLS */
-#include "zephyr/random/random.h"
-#include "tinycrypt/constants.h"
-#include "tinycrypt/ecc.h"
-#include "tinycrypt/ecc_dh.h"
 #endif /* CONFIG_MBEDTLS */
 
 #if defined(CONFIG_MBEDTLS)
```

Comment:
Since this test now is only built when CONFIG_MBEDTLS is set (after filters are set properly), I think we can remove all `#if defined(CONFIG_MBEDTLS)` from this file.